apps/arena/lib/arena/game_updater.ex

```diff
@@ -33,9 +33,9 @@ defmodule Arena.GameUpdater do
   # END API
   ##########################
 
-  def init(%{clients: clients}) do
+  def init(%{clients: clients, game_config: game_config}) do
```

Comment:
Why this change? Does it make sense for matchmaking to decide the config of a match?

Also, if we do separate matchmaking into its own app/server then this makes the communication more complex as we need to pass a configuration

Reply:
no it doesn't it was something to randomize the character from the bots, i'll restore this and try another approach, maybe hardcoding since this is a first iteration